README.md

```diff
@@ -185,6 +202,17 @@ The official way to find the ECR image URIs for your region is to use the SSM Pa
 aws ssm get-parameters-by-path --path /aws/service/aws-for-fluent-bit/
 ```
 
+#### Networking issue with Windows containers when using `async` DNS resolution by plugins
+There are Fluent Bit core plugins which are using the default [async mode for performance improvement](https://github.com/fluent/fluent-bit/blob/master/DEVELOPER_GUIDE.md#concurrency). Ideally when there are multiple nameservers present in a network namespace, the DNS resolver should retry with other nameservers if it receives failure response from the first one. However, Fluent Bit version 1.9 had a bug wherein the DNS resolution failed after first failure. The Fluent Bit Github issue link is [here](https://github.com/fluent/fluent-bit/issues/5862). 
+
+This is an issue for Windows containers running in `default` network where the first DNS server is the one used only for DNS resolution within container network i.e. resolving other containers connected on the network. Therefore, all other DNS queries fail with the first nameserver. 
+
+To work around this issue, we suggest using the following option so that system DNS resolver is used by Fluent Bit which works as expected.
+```
+net.dns.mode LEGACY
+```
+This workaround is applicable only for Fluent Bit 1.9 as of now.
```

Comment:
Sorry wait- do we know that its actually fixed in 2.0?

Until its fixed, we should probably not restrict this guidance to a version, since then it will become out of date

Reply:
Yes right. We have not validated the same in 2.0 till now. 

Let me remove this statement for now.